scripts/webpack/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export { default as resources } from './webpack-resources';
```

Comment:
this is necessary in order for TS to properly resolve workspace package imports

`import { resources } from @fluentui/scripts/webpack` etc...